src/scripts/components/VideoStatus.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import React, { Component } from 'react'
+// import styled from 'styled-components'
+// import Button from './foundations/Button'
+// import Input from './foundations/Input'
+import Wrapper from './foundations/Wrapper'
+
+type Props = {
+  video: Object,
+  videoFromPTI: Object
+};
+
+class VideoStatus extends Component<Props, void> {
+  render () {
+    let video = this.props.video
+    return (
+      <Wrapper>
+        <pre id="video-status">
+            Your Video is now being uploaded/beging transcoded/all ready for sharing!
+          <br />
+            Once it is ready,
+            You can share your video here [link] or embed it like this [link]
```

Comment:
Probably something we want to handle later, but we'll need to come up with a plan for internationalization